client/components/cart-confirmation.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Confirmation = props => {
+  return (
+    <div>
+      <h3>Pruchase is Processing</h3>
```

Comment:
typo in line 6 but we can fix that in another PR

Reply:
can I change the code on here or do i have to redo PR?